tests/sentry/api/endpoints/test_source_map_debug.py

```diff
@@ -336,10 +288,11 @@ def test_partial_url_match(self):
         assert error["message"] == "The absolute path url is a partial match"
         assert error["data"] == {
             "absPath": "https://app.example.com/static/static/js/application.js",
```

Comment:
Is there a reason why you use `/static/static` everywhere? Maybe reducing it to a single one will suffice?

Reply:
will change, i think it was from an example filepath I had from somewhere, but it is not needed